test/plugin/test_in_sql.rb

```diff
@@ -26,7 +26,7 @@ def teardown
       table messages
       tag logs
       update_column updated_at
-      time_column updated_at
+      time_column custom_time
```

Comment:
Add new test for `custome_time` is better.

Reply:
So I will happily do that if it's really what you want, but are you sure that's really the best solution? The point here is to test the SQL plugin, not Active Record's timestamp generation code. The reason I made this change was because the current test mostly wasn't testing the local timestamp generation logic...

Reply:
Created a9b6aa6 to address this if.